<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Sensor_table">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>41</y>
      <width>591</width>
      <height>391</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="lbl_sensor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensor Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Exit">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>551</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Refresh">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>388</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cb_Light">
       <property name="text">
        <string>light</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_Tem">
       <property name="text">
        <string>tem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_Hum">
       <property name="text">
        <string>hum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_Pre">
       <property name="text">
        <string>pre</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="cbb_Mac">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string notr="true">MacAdress</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>MacAdress</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_Mac">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Device:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_manager">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>디바이스 등록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Logout">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>로그아웃</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
